04/app.js

```diff
@@ -0,0 +1,27 @@
+function handleClick(event) {
+    event.stopPropagation();
```

Comment:
Raczej w tym miejscu nie powinno być zatrzymanie propagacji ;) - chodzi o to, aby kolory wypełniały po kolei div-y